engine/src/main/java/org/terasology/rendering/nui/layers/ingame/ExtraMenuScreen.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2019 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.nui.layers.ingame;
+
+import org.terasology.crashreporter.CrashReporter;
+import org.terasology.engine.GameEngine;
+import org.terasology.engine.LoggingContext;
+import org.terasology.engine.Time;
+import org.terasology.engine.modes.StateMainMenu;
+import org.terasology.network.NetworkMode;
+import org.terasology.network.NetworkSystem;
+import org.terasology.registry.CoreRegistry;
+import org.terasology.registry.In;
+import org.terasology.rendering.nui.CoreScreenLayer;
+import org.terasology.rendering.nui.WidgetUtil;
+import org.terasology.telemetry.TelemetryScreen;
+
+/**
+ * handles the extra menu button when game is paused
```

Comment:
This is pretty accurate, but could be written more nicely. Example:

```suggestion
 * Handles the "Extras" button from the game's pause menu screen.
```